k8sutils/statefulset.go

```diff
@@ -345,6 +345,24 @@ func getVolumeMount(name string, persistenceEnabled *bool, externalConfig *strin
 
 // getProbeInfo generate probe for Redis StatefulSet
 func getProbeInfo(probe *redisv1beta1.Probe) *corev1.Probe {
+	// when probe is nil, return default handler
```

Comment:
I believe we can define these default values using kubebuilders and crds.

https://book.kubebuilder.io/reference/markers/crd-validation.html